Mathlib/NumberTheory/DirichletCharacter/Basic.lean

```diff
@@ -96,23 +109,52 @@ lemma changeLevel_eq_cast_of_dvd {m : ℕ} (hm : n ∣ m) (a : Units (ZMod m)) :
 def FactorsThrough (d : ℕ) : Prop :=
   ∃ (h : d ∣ n) (χ₀ : DirichletCharacter R d), χ = changeLevel h χ₀
 
+lemma changeLevel_factorsThrough {m : ℕ} (hm : n ∣ m) : FactorsThrough (changeLevel hm χ) n :=
+  ⟨hm, χ, rfl⟩
+
 namespace FactorsThrough
 
+variable {χ}
+
 /-- The fact that `d` divides `n` when `χ` factors through a Dirichlet character at level `d` -/
 lemma dvd {d : ℕ} (h : FactorsThrough χ d) : d ∣ n := h.1
 
-/-- The Dirichlet character at level `d` through which `χ` factors -/
+/-- The Dirichlet character at level `d` through which `χ` factors. -/
```

Comment:
Quoting from the [Documentation style](https://leanprover-community.github.io/contribute/doc.html) page:
```quote
 If a doc string is a complete sentence, then it should end in a period.
```
Which I read as "a period is not required when the doc string is not a complete sentence". (But a period is certainly allowed in such a case.)